### DISCUSSION TITLE
Force floating window to layout

### DISCUSSION THREAD

**Discussion by nullbyto** [qtile/qtile#3722] [Q&A] [answered]
Hello, 

I'm new to `qtile` but I'm really liking it. Really good. Besides this issue #3680 its perfect.

Anyways, when windows are toggled to `floating` mode and then I switch the `layout`, the windows which are toggled stays floating, that's in some situations good, but I don't like it. Does anyone know how to force the windows to tile when changing layout or in main to toggle all? Instead of toggling every single one back.

Thanks.

**Comment by veksha** [comment, marked as answer]
try this:

```python
@hook.subscribe.layout_change
def _(layout, group):
	for window in group.windows:
		window.floating = False
```

**Reply by nullbyto** [reply]
Thank you works great.
